### ISSUE TITLE
Where can I log questions or issues related to the azure-devops-ui library?

### ISSUE THREAD

**Issue by ppejovic** [microsoft/azure-devops-extension-sample#48]
Not sure if this is the right forum for `azure-devops-ui` questions as that library doesn’t seem to be open source. 

I’m trying to load 100+ rows into a [`Table`](https://developer.microsoft.com/en-us/azure-devops/components/table) but am only getting the first 30 rows rendered. The only way I can seem to control the number of rows is using the `pageSize` property, but from the documentation that appears to be a property for controlling pre-rendering of rows and it is an arbitrary value that isn’t directly tied to row count. I don’t see a way to determine if the number of rows I provide to the table will actually be rendered in whole, even if that is a smaller number. Is there any way to achieve this?

I would like to implement paging or some type of lazy loading (like the “load more..” behaviour used elsewhere in Azure DevOps) but these don’t appear to be native features of the `Table` component. Am I missing anything or are there any examples of this? Or is this something where I’ll just have to roll my own implementation?

**Comment by karelkral** [comment]
Yes, my experience is that developing for AZ DevOps is a very frustrating thing. Nobody answers my questions, there is no forum, this repository is abandoned... 

And about Table - my experience is similar. Only 30 records are rendered. The documentation says nothing about that, By experimenting, I found that setting property pageSize to 100 resolves that. But I do not know why.

**Comment by soodputter** [comment]
I came across the same issue where trying to load data with scrolling but as said no proper documentation and the repository is abandoned. waiting fot the new release in september and hoping documentation will be appropriate this time.

**Comment by soodputter** [comment]
> Yes, my experience is that developing for AZ DevOps is a very frustrating thing. Nobody answers my questions, there is no forum, this repository is abandoned...
> 
> And about Table - my experience is similar. Only 30 records are rendered. The documentation says nothing about that, By experimenting, I found that setting property pageSize to 100 resolves that. But I do not know why.

Hi, did scroll load work for you? would be great if any sample you can provide.

**Comment by edtro** [comment]
> Yes, my experience is that developing for AZ DevOps is a very frustrating thing. Nobody answers my questions, there is no forum, this repository is abandoned...

Hi all,

This is also my experience... when you take a look at the offical pages of Formula Design System (ref: https://developer.microsoft.com/en-us/azure-devops/release-notes), you will notice that the last release was in **january 2020**.

So I really hope that Microsoft didn't abandon these libraries (and will switch over to GitHub entirely)... so our ongoing effort is not for nothing.
